ts/features/bonus/cgn/screens/merchants/CgnMerchantsTabsScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { Platform, SafeAreaView, StyleSheet } from "react-native";
+import { Tab, Tabs, View } from "native-base";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { Dispatch } from "../../../../../store/actions/types";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import I18n from "../../../../../i18n";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { availableMerchants } from "../../__mock__/availableMerchants";
+import { navigateToCgnMerchantDetail } from "../../navigation/actions";
+import customVariables from "../../../../../theme/variables";
+import { makeFontStyleObject } from "../../../../../theme/fonts";
+import CgnMerchantsListView from "../../components/merchants/CgnMerchantsListView";
+import { H1 } from "../../../../../components/core/typography/H1";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+export type TmpMerchantType = {
+  name: string;
+  category: string;
+  location: string;
+};
+
+const styles = StyleSheet.create({
+  tabBarContainer: {
+    elevation: 0,
+    height: 40
+  },
+  tabBarUnderline: {
+    borderBottomColor: customVariables.tabUnderlineColor,
+    borderBottomWidth: customVariables.tabUnderlineHeight
+  },
+  tabBarUnderlineActive: {
+    height: customVariables.tabUnderlineHeight,
+    // borders do not overlap each other, but stack naturally
+    marginBottom: -customVariables.tabUnderlineHeight,
+    backgroundColor: customVariables.contentPrimaryBackground
+  },
+  activeTextStyle: {
+    ...makeFontStyleObject(Platform.select, "600"),
```

Comment:
```suggestion
    ...makeFontStyleObject( "SemiBold"),
```
You should avoid to use deprecated code
Use instead `import { makeFontStyleObject } from "../../../../../components/core/fonts";`